app/serializers/lists/protocol_serializer.rb

```diff
@@ -6,13 +6,25 @@ class ProtocolSerializer < ActiveModel::Serializer
     include Rails.application.routes.url_helpers
 
     attributes :name, :code, :keywords, :linked_tasks, :nr_of_versions, :assigned_users, :published_by,
-               :published_on, :updated_at, :archived_by, :archived_on, :urls, :default_public_user_role_id,
-               :hidden, :top_level_assignable, :has_draft, :team, :permissions
+               :published_on, :updated_at, :archived_by, :archived_on, :urls,
+               :top_level_assignable, :has_draft, :team, :permissions
+    attribute :hidden
```

Comment:
Not sure that you need define each attribute manually, they can be in general list as before